test/Runtime/CMakeLists.txt

```diff
@@ -1,39 +1,15 @@
-#===-- test/Runtime/CMakeLists.txt -----------------------------------------===#
```

Comment:
Why remove?  Other projects include the LLVM license header in CMakeList.txt files?

Reply:
For what it is worth, the bulk of these changes come from Pat's change.

The assertion that LLVM CMakeLists.txt files include the license banner is false. So I assume that is why Pat has removed them from most of the files he touched in his PR.

Reply:
I basically just merged all of his changes and then went through and resolved comments and fixed major problems to this point. I still did not fix the lit test problem. But I need to do that ASAP.